### PR TITLE
Robuster waiting for validators' readiness

### PR DESCRIPTION
**Pull request by adaszko**
https://rchain.atlassian.net/browse/RCHAIN-1459

### CODE REVIEW

File: ci/is-container-log-line-present.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -ue
+
+
+main () {
+    if [[ $# -ne 2 ]]; then
+        echo "$0: wrong number of arguments; expected 2, got $#"
+        return 2
+    fi
+    local container_name_substring=$1; shift
+    local log_message=$1; shift
```
Comment (lukasz-golebiewski) [inline ci/is-container-log-line-present.sh:10]: What's the benefit of `shift` over `$2`?

File: .drone.yml [hunk 2]
```diff
@@ -84,13 +84,19 @@ pipeline:
     commands:
       - /opt/docker/bin/rnode -c ./drone/config/bootstrap.toml run
 
+  wait-for-bootstrap:
+    image: docker
+    commands:
+      - until ci/is-container-log-line-present.sh _step_4 'Making a transition to ApprovedBlockRecievedHandler state.'; do echo $?; sleep 5; done
```
Comment (lukasz-golebiewski) [inline .drone.yml:90]: Does drone yaml version support constants? `Making a transition to ApprovedBlockRecievedHandler state.` is a good candidate

Comment (adaszko) [reply .drone.yml:90]: I don't believe it does

File: ci/is-container-log-line-present.sh (hunk 1 above)
Comment (adaszko) [reply ci/is-container-log-line-present.sh:10]: Adding and removing parameters generates fewer mistakes

**Review by lukasz-golebiewski — APPROVED** [review]

**Pull request merged**